yw_website/yw_rest_services/views.py

```diff
@@ -61,8 +61,43 @@ def create_workflow(request):
         'run': rdata
         })
 
-    
- 
-
+@api_view(['post'])
+@permission_classes((permissions.AllowAny,))
+def update_workflow(request, workflow_id):
+    username = User.objects.filter(username=request.data.get('username', None))
+    if not username:
+        return Response(status=500, data={'error':'bad username'})
+    username = username[0]
 
+    try:
+        w = Workflow.objects.get(pk=workflow_id)
+    except Workflow.DoesNotExist:
+        return Response(status=500, data={'error':'workflow does not exist'})
 
+    v = Version(
+        workflow=Workflow.objects.get(pk=w.id),
```

Comment:
we want to assign an id to workflow, not a workflow object with the primary key of w.id

Reply:
No we should not do that incase we change the primary key of the model. Also that is not how the Django examples were structured.

Reply:
again, scratch that. Forgot how the ORM worked. This is good